x-pack/plugins/reporting/native_controller/child_process.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
```

Comment:
The eslint license header rules are probably to blame for all these empty lines, but if you want to clean them up I wouldn't be opposed :)